dist/phila-vue-datafetch.js

```diff
@@ -5331,12 +5331,14 @@
                      return object._featureId === state.activeFeature.featureId
                      });
       } else {
-        input.push(state.geocode.data);
-        for (var i = 0, list = state.geocode.related; i < list.length; i += 1) {
-          var relate = list[i];
-
-            input.push(relate);
+        var data;
```

Comment:
prefer let over var

Reply:
Same, I rarely use it but occasionally I will use it depending on the scope.